.github/workflows/test-node.yaml

```diff
@@ -22,3 +22,37 @@ jobs:
         run: |
           npm install prettier --prefix ./pkg/new-ui/v1beta1/frontend
           make prettier-check
+
+  frontend-unit-tests:
+    name: Frontend Unit Tests
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.18.1
+      
+      - name: Fetch Kubeflow and install common code dependencies
+        run: |
+          cd /tmp && git clone https://github.com/kubeflow/kubeflow.git
+          cd kubeflow
+          git checkout c4ca7a9
+          cd components/crud-web-apps/common/frontend/kubeflow-common-lib
+          npm i
+          npm run build
+          npm link ./dist/kubeflow
+
+        - name: Install KWA dependencies
+          run: |
+            cd pkg/new-ui/v1beta1/frontend
+            npm i
+            npm link kubeflow
+
+        - name: Run unit tests
+          run: |
+            cd katib/pkg/new-ui/v1beta1/frontend
+            npm run test
```

Comment:
Can you remove one of the top tabs since these items maybe are wrong place?

Reply:
Fixed this! Thank you @tenzen-y